modules/nf-lineage/src/main/nextflow/lineage/model/DataOutput.groovy

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2013-2025, Seqera Labs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package nextflow.lineage.model
+
+import groovy.transform.Canonical
+import groovy.transform.CompileStatic
+import nextflow.lineage.serde.LinSerializable
+
+import java.time.OffsetDateTime
+
+/**
+ * Model a base class for workflow and task outputs
+ *
+ * @author Paolo Di Tommaso <paolo.ditommaso@gmail.com>
+ */
+@Canonical
+@CompileStatic
+class DataOutput implements LinSerializable {
+    /**
```

Comment:
Is this class used for file outputs? If so then I would call it `FileOutput`, I'm having a hard time understanding what it's used for

Reply:
This is used to describe the published files. They can be files or dirs, it is the reason why I called data but  no problem to change to file to be more coherent with the rest of the code.